src/main/cms/cms_menu_quick.c

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
These are the old licence files. Please use the default licence md content from the root directory of the repo.

Reply:
Done! Thanks!